addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,7 +10976,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot and episode thumbnail for unwatched items"
```

Comment:
This change makes it only seem like it's for episodes and not for movies like the help string suggests. Also it's way too long.